lib/Ohmbrewer_Onewire.cpp

```diff
@@ -3,16 +3,18 @@
 #include "ds18x20.h"
 #include "onewire.h"
 
+
 /**
  * Constructors
  * @param busPin Digital pin that serves as the bus for all onewire
  * @param probeId Unique ID for the temperature probe [8] char array ID code
  */
 Ohmbrewer::Onewire::Onewire() : Ohmbrewer::Probe(){
-//    ow_setPin(D0);
+    Serial.println("Onewire");
```

Comment:
You can clear out the debugging statements now that it's working (or at least comment them out).